src/Symfony/Component/Validator/Resources/translations/validators.pl.xlf

```diff
@@ -192,7 +192,7 @@
             </trans-unit>
             <trans-unit id="51">
                 <source>No temporary folder was configured in php.ini.</source>
-                <target>Nie skonfigurowano folderu tymczasowego w php.ini, lub skonfigurowany folder nie istnieje.</target>
+                <target>Nie skonfigurowano folderu tymczasowego w php.ini lub skonfigurowany folder nie istnieje.</target>
             </trans-unit>
```

Comment:
You hardly ever put comma before `or` in Polish. Am I right @jakzal?

Reply:
I’m no expert but it looks weird here.

Reply:
Would you like me to ping another Polish native speaker? By the way, sorry for bothering you, but you were the first Polish speaking contributor, that came to my mind.

Reply:
I am Polish native speaker and i also think this comma is unnecessary.